internal/beef/beef_test.go

```diff
@@ -3,11 +3,10 @@ package beef
 import (
 	"encoding/hex"
 	"errors"
+	"github.com/bitcoin-sv/go-sdk/script"
+	"github.com/bitcoin-sv/go-sdk/transaction"
```

Comment:
nit: move these imports below next to the other 3rd party dependencies

Reply:
Done, refactored all imports by gci linter